packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -251,6 +252,7 @@ export type FilterMap = Map<
 >;
 export interface IrisGridProps {
   children: React.ReactNode;
+  dh: DhType;
```

Comment:
Side note - it would be ideal if everything was abstracted through the model and we didn't have to depend on `dh` being passed in to `IrisGrid`. I think that change is much more work than we're looking for right now though with this change.

Also, we could still use the context in the class component to get the API from the context: https://legacy.reactjs.org/docs/context.html#classcontexttype

I don't think that's much better though.